test/common.jl

```diff
@@ -0,0 +1,29 @@
+using ZipArchives: 
```

Comment:
Does it make sense to allow blind `using` in the tests to check exports?
Otherwise the internal changes look good to me!

Reply:
Thanks for the suggestion. Any idea what kind of errors a blind `using` would catch? Also, I guess I should add tests on the latest stable Julia version instead of just 1.6.